my-react-app/src/pages/notfound/NotFoundPage.tsx

```diff
@@ -0,0 +1,15 @@
+import { PageTitleProps } from 'models/models';
+import React, { useEffect } from 'react';
+import './NotFoundPage.css';
+
+export function NotFoundPage(props: PageTitleProps) {
+  useEffect(() => {
+    props.setTitle('Not found page');
+  });
+
+  return (
+    <div>
+      <h1 className="page-header">Not found page</h1>
```

Comment:
```suggestion
      <h1 className="page-header">404 Page Not Found</h1>
```
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404